test-fixtures/build.gradle

```diff
@@ -0,0 +1,51 @@
+dependencies {
```

Comment:
almost feels like this should be forked out of the API into a testing library but it's probably not worth it :)

Reply:
This has always been a principle of the sonar-plugin-api to provide utility classes to help testing plugins. I think moving test utils to a separate artifact that won't be available in production is already a good step. :)

Reply:
Agreed!